infra/reachability-metadata/src/main/resources/META-INF/native-image/org.apache.shardingsphere/generated-reachability-metadata/resource-config.json

```diff
@@ -2794,8 +2782,5 @@
   "bundles":[{
     "name":"com.microsoft.sqlserver.jdbc.SQLServerResource",
     "locales":["zh-CN"]
-  }, {
-    "name":"sun.text.resources.cldr.FormatData",
-    "locales":["en"]
   }]
 }
```

Comment:
@linghengqian Should we keep blank line at the end of JSON file?

Reply:
- This is what the GraalVM Tracing Agent ( under GraalVM CE For JDK 22.0.2 ) does after executing `./mvnw -PgenerateMetadata -DskipNativeTests -e -T1C clean test native:metadata-copy` to fix nativeTest. 

- As discussed earlier, the entire JSON file in the same folder is automatically generated and is considered to be unchangeable by https://shardingsphere.apache.org/document/current/cn/user-manual/shardingsphere-jdbc/graalvm-native-image/ . CC @terrymanu .

- Do you think we need to add checkstyle or spotless rules to automatically detect and add blank lines?

Reply:
I'm just curious if this is an omission. If it is confirmed to be unchangeable, we can just keep it as it is.